osu.Game/Overlays/Profile/Sections/PaginatedProfileSubsection.cs

```diff
@@ -116,16 +116,14 @@ private void showMore()
 
             CurrentPage = CurrentPage?.TakeNext(ItemsPerPage) ?? new PaginationParameters(InitialItemsCount);
 
-            retrievalRequest = CreateRequest(User.Value, CurrentPage.Value);
+            retrievalRequest = CreateRequest(User.Value, new PaginationParameters(CurrentPage.Value.Offset, CurrentPage.Value.Limit + 1));
```

Comment:
If this is taking the limit from the previous page every time, doesn't this mean that the number of fetched items increases with *every request*? (think fetching 6 items, then 7, 8, 9...)

Probably should be `InitialItemsCount + 1` or something.